### PR TITLE
Don't check remote sources for trickplay

### PR DESCRIPTION
**Pull request by gnattu**
These sources will not have trickplay generated anyway.

**Changes**

**Issues**

### CODE REVIEW

**Review by JPVenson — APPROVED** [review]

**Review by nyanmisaka — APPROVED** [review]

File: Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs [hunk 1]
```diff
@@ -498,6 +498,11 @@ public async Task<Dictionary<string, Dictionary<int, TrickplayInfo>>> GetTrickpl
         var trickplayManifest = new Dictionary<string, Dictionary<int, TrickplayInfo>>();
         foreach (var mediaSource in item.GetMediaSources(false))
         {
+            if (mediaSource.IsRemote)
```
Comment (JPVenson) [inline Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:501]:
```suggestion
            if (mediaSource.IsRemote || !Guid.TryParse(mediaSource.Id, out var mediaSourceId)
            {
                continue;
            }
```

File: Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs [hunk 2]
```diff
@@ -498,6 +498,11 @@ public async Task<Dictionary<string, Dictionary<int, TrickplayInfo>>> GetTrickpl
...
+            {
+                continue;
+            }
+
             var mediaSourceId = Guid.Parse(mediaSource.Id);
```
Comment (JPVenson) [inline Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:506]:
```suggestion
            if (mediaSource.IsRemote || !Guid.TryParse(mediaSource.Id, out var mediaSourceId)
            {
                continue;
            }
```

Comment (gnattu) [reply Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:506]: Do we really want to do this?

Comment (JPVenson) [reply Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:506]: dont see harm in just checking for it.
Could add a log debug entry as well but its better then just failing.

Comment (JPVenson) [reply Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:506]: better would be to make the Id field non nullable/required but thats a bit outside the scope

**Review by Bond-009 — APPROVED** [review]

**Review by crobibero — APPROVED** [review]

**Pull request merged**